### ISSUE TITLE
Can someone help me with running Grant's new codes with manimGL? Many thanks.

### ISSUE THREAD

**Issue by Lay-Yin** [3b1b/videos#72]
I've installed manimGL version 1.6.1 along with the necessary dependencies listed in requirement.txt to virtual env(manimgl_test).As you can see from figures, the manimgl directory structures are identical.
![image](https://github.com/3b1b/videos/assets/131653407/3a640077-1d07-4289-9472-53540a82cb57)
![image](https://github.com/3b1b/videos/assets/131653407/20ec52e6-785e-4b19-829c-8573279bb310)

When I try to run the simple Py code(test.py), I got this result (figure 3). 
![image](https://github.com/3b1b/videos/assets/131653407/1ebf01d2-4b2f-45c6-b306-502a6703a011)

Which could be rendered perfectly in manimgl(Another virtual env, where I installed manimGL version 1.6.1 by git clone command. Of which the directory structure is a little different from grant's. As shown in figure below.
![image](https://github.com/3b1b/videos/assets/131653407/5c42e94c-96e7-4a50-9d05-15e211c8a6c1)

Later, after downloading the 2023_optics_puzzles folder, when I try to execute one of the py.files (bending_waves.py), things just don't work out, like what figures down below shows.
![image](https://github.com/3b1b/videos/assets/131653407/78f770de-6207-4289-80a3-ec36178aadb9)
![image](https://github.com/3b1b/videos/assets/131653407/98dd450c-c058-4b2a-87af-04ebd94e243b)

In a nutshell,  virtual env manimgl seems to be better, after all test.py can be executed. But Grant's codes are more incompatible with it, since more bugs reported than excuted in manimgl_test virtual env.

**Comment by Lay-Yin** [comment]
Any opinion would be great opinion！
